src/android-service.sh

```diff
@@ -33,8 +34,14 @@ start() {
 		android_start ${service_service}
 	fi
 
+	if [ -e /etc/systemd/android/${service_service} ]; then
+		source /etc/systemd/android/${service_service}
```

Comment:
I think we can use a drop in for that, what do you think?

something like

```
[Service]
Environment=WAIT_TIMEOUT=120
```

and then if not set, default to 60. What do you think?

Reply:
Yes, make sense :)